sbol_utilities/sbol3_genbank_conversion.py

```diff
@@ -58,7 +62,7 @@ class GenBank_SBOL3_Converter:
     gb2so_map = {}
     so2gb_map = {}
     default_SO_ontology = "SO:0000110"
-    default_GB_ontology = ""  # TODO: Whats the default here?
+    default_GB_ontology = "todo"  # TODO: Whats the default here?
```

Comment:
Right name for these is "term", not "ontology" - the ontology is the full collection of organized terms.

Reply:
Oh, alright, I changed them.